src/main/java/duke/util/Ui.java

```diff
@@ -0,0 +1,20 @@
+package duke.util;
+
+public class Ui {
+    //Duke logo
+    private static final String logo = " ____        _        \n"
+            + "|  _ \\ _   _| | _____ \n"
+            + "| | | | | | | |/ / _ \\\n"
+            + "| |_| | |_| |   <  __/\n"
+            + "|____/ \\__,_|_|\\_\\___|\n";
+
+    public Ui() {
+    }
+
+    /**
+     * Welcome statement for new user.
+     */
+    public void welcome() {
```

Comment:
Perhaps your method name could be more descriptive here? For example, change `welcome` to `displayWelcomeMsg`.

Reply:
Seconded.